.bazelbuilderversion

```diff
@@ -0,0 +1 @@
+cockroachdb/bazel:20220328-163955
```

Comment:
Nit: Can we put this file somewhere else besides top-level? In the `build` directory seems to make sense.

Reply:
Done